src/Elastic.Markdown/Myst/CodeBlocks/EnhancedCodeBlockHtmlRenderer.cs

```diff
@@ -107,6 +107,22 @@ private static int CountIndentation(StringSlice slice)
 		return indentCount;
 	}
 
+	private static bool IsCommentBlock(Block block)
+	{
+		if (block.GetType().Name == "CommentBlock")
```

Comment:
nit: 
```suggestion
		if (block is CommentBlock)
```
(just for consistency)

Reply:
Hmm. It looks like this results in:

```
error CS0246: The type or namespace name 'CommentBlock' could not be found (are you missing a using directive or an assembly reference?)
```

Reply:
Reverted for now as I don't know how to fix this code so that `if (block is CommentBlock)` works. Open to any suggestions!

Reply:
Need to add `using Elastic.Markdown.Myst.Comments;`  that will allow the `is` check